src/components/DepositFee/index.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react'
+import Tippy from '@tippyjs/react'
+import { isAddress } from 'ethers'
+import { useConnectWallet } from '@web3-onboard/react'
+
+import Button from '../Button'
+import InputText from '../Inputs/InputText'
+import InputRange from '../Inputs/InputRange'
+import InputToggle from '../Inputs/InputToggle'
+
+import * as S from './styles'
+
+type FeesData = {
+  isChecked: boolean
+  feeRate?: string
+  brokerCommision?: number
+  managerShare?: number
+}
+
+interface IDepositFeeProps {
+  feesData?: Record<string, FeesData>
+  disabledNoEvent?: boolean
```

Comment:
What does this variable do? Is it the best name you can give it?

I'm saying that outside the context of what you're building, without reading through the code, I don't get what that is supposed to do.

After looking at the Button component, I also saw a problem:
![image](https://github.com/KassandraFinance/kassandra-application/assets/86835927/2a090c05-35d5-4e0f-a42d-03a646dae9c5)

You have a prop `disabledNoEvent` that is doing the same purpose of the `disabled` prop, you should probably remove this `disabledNoEvent` prop and use only `disabled`, it will also be more readable.